src/main/resources/templates/test_assigned.ftlh

```diff
@@ -0,0 +1,9 @@
+<html>
+<head>
+    <title>Test mail!</title>
+</head>
+<body>
+<h1>Hello, ${userName}!</h1>
+<p>Test is assigned for you</p>
```

Comment:
Seams like a test value. Are you going to move creating templates to a separate task?

Reply:
I'm going to create templates for each notification type a bit later :)